core/domain/user_services.py

```diff
@@ -1870,16 +1870,15 @@ def allow_user_to_review_translation_in_language(user_id, language_code):
     Args:
         user_id: str. The unique ID of the user.
         language_code: str. The code of the language. Callers should ensure that
-            the user does not have rights to review translations in the given
+            the user does not has rights to review translations in the given
```

Comment:
has --> have. The previous version was correct. The "singular form" is already there in "does not".

Reply:
Done! (Thanks for the explanation. :) )